packages/components/Avatar/src/useAvatar.ts

```diff
@@ -161,6 +163,29 @@ export const removeTitlesFromName = (words: string[]): string[] => {
   return words.filter((word) => !titles.has(word));
 };
 
+/**
+ * A function which verifies whether words array contains multi-word title
+ *
+ * @param name
+ * @returns name without multi-word titles
+ */
+const removeMultiWordTitles = (name: string, idx: number = 0): string => {
+  if (!isMultiWordName || idx >= multiWordTitles.length) return name;
```

Comment:
I think isMultiWordName here is checking for the existance of the isMultiWordName function, not checking that the name is multi word.

Reply:
With new implementation this check is not needed here. It was added for the cases when there's more then one multi-word title

Reply:
Yeah, I see, I didn't call the function. Thanks for catching this